arbor/cable_cell_param.cpp

```diff
@@ -152,4 +152,37 @@ void decor::set_default(defaultable what) {
             what);
 }
 
+std::ostream& operator<<(std::ostream& os, const paintable& item) {
+    std::visit(
+        [&] (const auto& p) {
+            using T = std::decay_t<decltype(p)>;
+            if constexpr (std::is_same_v<init_membrane_potential, T>) {
+                os << "init-membrane-potential";
```

Comment:
We have been printing everything in the decor part of the code as s-expressions, so maybe it's worth sticking to that style? Also we could print the value while we're at it.
Something like `(init-membrane-potential -65)`.

Reply:
I deliberated on the same question, here's what I swayed me against
1. telling _what_ might be enough information 
2. ...and not overloading the reader, the line is quite long already.
3. also, where to draw the line? `(density ('name' (...)))`? Or print

Reply:
Fair enough. Should we keep it out of the public header as it's not printing the "full" information about the class?

Reply:
Ok, good point to think about. It just prints the outer DSL constructor in s-exp-speak.
At some point we had opened an issue on a 'general object printing facility', maybe it's
a good time to take that a bit seriously?

For now, I'll change `<<` to `print_paintable` and hide it.